packages/auth-compat/demo/public/script.js

```diff
@@ -446,6 +446,31 @@ function onSignInWithCustomToken(event) {
     .then(onAuthUserCredentialSuccess, onAuthError);
 }
 
+/**
+ * Sets a custom token provider that simply returns the custom token provided
+ * in the #user-custom-token-for-provider field.
+ * @param {DOMEvent} _event HTML DOM event returned by the listener.
+ */
+function onSetCustomTokenProvider(event) {
+  const nextRefreshToken = $('#user-custom-token-for-provider').val();
```

Comment:
nit: `nextCustomToken` maybe? Since this is not a refresh token.

Reply:
Whoops, good point, thank you!